libraries/APM_Control/AP_PitchController.cpp

```diff
@@ -158,6 +159,15 @@ const AP_Param::GroupInfo AP_PitchController::var_info[] = {
 
     AP_SUBGROUPINFO(rate_pid, "_RATE_", 11, AP_PitchController, AC_PID),
 
+    // @Param: 2SRV_FLAP
+    // @DisplayName: Flap compensation
+    // @Description: Gain added to pitch to keep aircraft from descending or ascending when deploying flaps. Decrease to compensate for pitching up when deploying flaps, increase if the vehicle pitches down when deploying flaps.
```

Comment:
I think what it is actually doing is changing the elevator trip with flaps. I think this could be done a lot more simply in Plane::stabilize_pitch() with the parameter in the main plane code.